tests/simpleharness/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Per offline conversation, we'll remove this for now.  Let's file an issue to make sure that desktop processes get elevated properly.